packages/flutter_tools/lib/src/isolated/native_assets/native_assets.dart

```diff
@@ -4,6 +4,8 @@
 
 // Logic for native assets shared between all host OSes.
 
+import 'dart:io' as io; // flutter_ignore: dart_io_import
```

Comment:
Maybe add the comment here?

Reply:
I prefer not repeating the comment below here.

A `git blame` will easily identify this commit where one can see the `WARNING` I added below.